### PR TITLE
Updated  tensorflow/tensorflow/lite/python/lite.py with grammatical a…

### PR DESCRIPTION
**Pull request by LakshmiKalaKadali**
Updated  tensorflow/tensorflow/lite/python/lite.py with grammatical and typo corrections. The TFLiteConverter API link:https://www.tensorflow.org/api_docs/python/tf/lite/TFLiteConverter

### CODE REVIEW

**Review by gbaned — APPROVED** [review]

File: tensorflow/lite/python/lite.py [hunk 1]
```diff
@@ -1437,11 +1437,11 @@ def convert(self, graph_def, input_tensors, output_tensors):
       output_tensors: List of output tensors.
 
     Returns:
-      The converted data in serialized format.
+      The converted data is in serialized format.
```
Comment (mihaimaruseac) [inline tensorflow/lite/python/lite.py:1440]:
```suggestion
      The converted data in serialized format.
```

Please revert this edit. The format of `Returns:` is a description of the returned value, as a noun phrase. It should be read as "this function returns the converted data in serialized format", but with your edit it gets to "this function returns the converted data is in serialized format"

File: tensorflow/lite/python/lite.py [hunk 2]
```diff
@@ -1519,11 +1519,11 @@ def convert(self):
     """Converts a TensorFlow GraphDef based on instance variables.
 
     Returns:
-      The converted data in serialized format.
+      The converted data is in serialized format.
```
Comment (mihaimaruseac) [inline tensorflow/lite/python/lite.py:1522]:
```suggestion
      The converted data in serialized format.
```

Same here

File: tensorflow/lite/python/lite.py [hunk 3]
```diff
@@ -1714,7 +1714,7 @@ def _convert_as_saved_model(self):
     """Converts a Keras model as a saved model.
 
     Returns:
-      The converted data in serialized format.
+      The converted data is in serialized format.
```
Comment (mihaimaruseac) [inline tensorflow/lite/python/lite.py:1717]:
```suggestion
      The converted data in serialized format.
```

File: tensorflow/lite/python/lite.py [hunk 4]
```diff
@@ -1733,7 +1733,7 @@ def convert(self):
     """Converts a keras model based on instance variables.
 
     Returns:
-      The converted data in serialized format.
+      The converted data is in serialized format.
```
Comment (mihaimaruseac) [inline tensorflow/lite/python/lite.py:1736]:
```suggestion
      The converted data in serialized format.
```

File: tensorflow/lite/python/lite.py [hunk 5]
```diff
@@ -1889,7 +1889,7 @@ def _convert_as_saved_model(self):
     """Converts the given concrete functions as a saved model format.
 
     Returns:
-      The converted data in serialized format.
+      The converted data is in serialized format.
```
Comment (mihaimaruseac) [inline tensorflow/lite/python/lite.py:1892]:
```suggestion
      The converted data in serialized format.
```

File: tensorflow/lite/python/lite.py [hunk 6]
```diff
@@ -1908,11 +1908,11 @@ def convert(self):
     """Converts a TensorFlow GraphDef based on instance variables.
 
     Returns:
-      The converted data in serialized format.
+      The converted data is in serialized format.
```
Comment (mihaimaruseac) [inline tensorflow/lite/python/lite.py:1911]:
```suggestion
      The converted data in serialized format.
```

File: tensorflow/lite/python/lite.py [hunk 7]
```diff
@@ -1977,7 +1977,7 @@ def convert(self):
     """Converts a Jax serving func based on instance variables.
 
     Returns:
-      The converted data in serialized format.
+      The converted data is in serialized format.
```
Comment (mihaimaruseac) [inline tensorflow/lite/python/lite.py:1980]:
```suggestion
      The converted data in serialized format.
```

File: tensorflow/lite/python/lite.py [hunk 8]
```diff
@@ -2315,11 +2315,11 @@ def convert(self):
     """Converts a TensorFlow GraphDef based on instance variables.
 
     Returns:
-      The converted data in serialized format.
+      The converted data is in serialized format.
```
Comment (mihaimaruseac) [inline tensorflow/lite/python/lite.py:2318]:
```suggestion
      The converted data in serialized format.
```

File: tensorflow/lite/python/lite.py [hunk 9]
```diff
@@ -2511,8 +2511,8 @@ def convert(self):
     """Converts a TensorFlow GraphDef based on instance variables.
 
     Returns:
-      The converted data in serialized format. Either a TFLite Flatbuffer or a
-      Graphviz graph depending on value in `output_format`.
+      The converted data is in serialized format. Either a TFLite Flatbuffer or
```
Comment (mihaimaruseac) [inline tensorflow/lite/python/lite.py:2514]:
```suggestion
      The converted data in serialized format, either a TFLite Flatbuffer or
```

File: tensorflow/lite/python/lite.py [hunk 10]
```diff
@@ -2734,8 +2734,8 @@ def convert(self):
     (`interpreter.get_signature_runner`).
 
     Returns:
-      The converted data in serialized format. Either a TFLite Flatbuffer or a
-      Graphviz graph depending on value in `output_format`.
+      The converted data is in serialized format. Either a TFLite Flatbuffer or
```
Comment (mihaimaruseac) [inline tensorflow/lite/python/lite.py:2737]:
```suggestion
      The converted data in serialized format, either a TFLite Flatbuffer or
```

File: tensorflow/lite/python/lite.py [hunk 11]
```diff
@@ -2862,7 +2862,7 @@ def _convert_as_saved_model(self):
     """Converts a Keras model as a saved model.
 
     Returns:
-      The converted data in serialized format.
+      The converted data is in serialized format.
```
Comment (mihaimaruseac) [inline tensorflow/lite/python/lite.py:2865]:
```suggestion
      The converted data in serialized format.
```

File: tensorflow/lite/python/lite.py [hunk 12]
```diff
@@ -2877,8 +2877,8 @@ def convert(self):
     """Converts a Keras model based on instance variables.
 
     Returns:
-      The converted data in serialized format. Either a TFLite Flatbuffer or a
-      Graphviz graph depending on value in `output_format`.
+      The converted data is in serialized format. Either a TFLite Flatbuffer or
```
Comment (mihaimaruseac) [inline tensorflow/lite/python/lite.py:2880]:
```suggestion
      The converted data in serialized format, either a TFLite Flatbuffer or
```

File: tensorflow/lite/python/lite.py [hunk 13]
```diff
@@ -2955,8 +2955,8 @@ def convert(self):
     """Converts a TensorFlow GraphDef based on instance variables.
 
     Returns:
-      The converted data in serialized format. Either a TFLite Flatbuffer or a
-      Graphviz graph depending on value in `output_format`.
+      The converted data is in serialized format. Either a TFLite Flatbuffer or
```
Comment (mihaimaruseac) [inline tensorflow/lite/python/lite.py:2958]:
```suggestion
      The converted data in serialized format, either a TFLite Flatbuffer or
```

File: tensorflow/lite/python/lite.py [hunk 14]
```diff
@@ -3340,8 +3340,8 @@ def convert(self):
...
```
Comment (mihaimaruseac) [inline tensorflow/lite/python/lite.py:3343]:
```suggestion
      The converted data in serialized format, either a TFLite Flatbuffer or
```

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

**Review by mihaimaruseac — APPROVED** [review]

File: tensorflow/lite/python/lite.py [hunk 15]
```diff
@@ -2407,7 +2407,7 @@ def _validate_quantized_input_stats(self, converter_kwargs, quant_mode):
       raise ValueError(
           "The `quantized_input_stats` flag must be defined when either "
           "`inference_type` flag or `inference_input_type` flag is set to "
-          "tf.int8 or tf.uint8. Currently, `inference_type={}` and "
+          "tf.int8 or tf.uint8. Currently `inference_type={}` and "
```
Comment (mihaimaruseac) [inline tensorflow/lite/python/lite.py:2410]:
```suggestion
          "tf.int8 or tf.uint8. Currently, `inference_type={}` and "
```

Comma was correct here, removing it causes tests to fail

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

**Review by mihaimaruseac — APPROVED** [review]

**Pull request merged**